METRICS.md

```diff
@@ -70,47 +77,54 @@ memory_stats/swap_usage/failcnt | uint64 | The number of times the swap space li
+memory_stats/statistics/unevictable | uint64 | The number of bytes of memory that cannot be reclaimed
+memory_stats/statistics/working_set | uint64 | The total number of bytes of memory that is being used and not easily dropped by the kernel
+memory_stats/statistics/writeback | uint64 | The number of bytes of file/anon cache that are queued for syncing to disk
+memory_stats/statistics/total_active_anon | uint64 | The total number of bytes of anonymous and swap cache memory on active LRU list <sup>(1)</sup>
+memory_stats/statistics/total_active_file | uint64 | The total number of bytes of file-backed memory on active LRU list <sup>(1)</sup>
+memory_stats/statistics/total_cache | uint64 | The total number of bytes of page cache memory <sup>(1)</sup>
+memory_stats/statistics/total_dirty | uint64 | The total number of bytes that are waiting to get written back to the disk <sup>(1)</sup>
+memory_stats/statistics/total_inactive_anon | uint64 | The total number of bytes of anonymous and swap cache memory on inactive LRU list <sup>(1)</sup>
+memory_stats/statistics/total_inactive_file | uint64 | The total number of bytes of file-backed memory on inactive LRU list <sup>(1)</sup>
+memory_stats/statistics/total_mapped_file | uint64 | The total number of bytes of mapped file (includes tmpfs/shmem) <sup>(1)</sup>
+memory_stats/statistics/total_pgfault | uint64 | The total number of page faults which happened since the creation of the cgroup <sup>(1)</sup>
+memory_stats/statistics/total_pgmajfault | uint64 | The total number of page major faults which happened since the creation of the cgroup <sup>(1)</sup>
+memory_stats/statistics/total_pgpgin | uint64 | The total number of charging events to the memory cgroup <sup>(1)</sup>
+memory_stats/statistics/total_pgpgout | uint64 | The total number of uncharging events to the memory cgroup <sup>(1)</sup>
+memory_stats/statistics/total_rss | uint64 | The total number of bytes of anonymous and swap cache memory <sup>(1)</sup>
+memory_stats/statistics/total_rss_huge | uint64 | The total number of bytes of anonymous transparent hugepages <sup>(1)</sup>
+memory_stats/statistics/total_swap | uint64 | The total number of bytes of swap usage <sup>(1)</sup>
+memory_stats/statistics/total_unevictable | uint64 | The total number of bytes of memory that cannot be reclaimed <sup>(1)</sup>
+memory_stats/statistics/total_writeback | uint64 | The total number of bytes of file/anon cache that are queued for syncing to disk <sup>(1)</sup>
+| |
+hugetlb_stats/\<size\>/failcnt | uint64 | The number of allocation failure due to HugeTLB limit
+hugetlb_stats/\<size\>/max_usage | uint64 | Max "hugepagesize" hugetlb  usage recorded
+hugetlb_stats/\<size\>/usage | uint64 | Current usage for "hugepagesize" hugetlb
+| |
+blkio_stats/io_merged_recursive/\<N\>/value | uint64 | Total number of bios/requests merged into requests belonging from all the descendant cgroups 
```

Comment:
On the occasion if you could remove metric "network/name" which is no longer exposed (because of including name as a dynamic element), it will be great :)